.oxlintrc.json

```diff
@@ -8,10 +12,15 @@
 	"rules": {
 		"max-lines-per-function": "off",
 		"max-lines": "off",
-		"max-depth": "off"
+		"max-depth": "off",
+		"eqeqeq": ["error", "always", { "null": "ignore" }]
 	},
-	"env": {
-		"browser": true,
-		"node": true
-	}
+	"overrides": [
```

Comment:
Next automatically adds a new line to `next-env.d.ts` when executing `pnpm run dev`, leading to an OXLint error. The file itself states it should not be edited manually.

Therefore I excluded it from the `triple-slash-reference` rule.

Reply:
#89 adds the file to oxlint ignore list.